src/ocamlorg_frontend/pages/package_overview.eml

```diff
@@ -54,14 +54,14 @@ in
       Install
   </h2>
 
-  <%s! Copy_to_clipboard.small_code_snippet ~id:"install-package" ~class_:"whitespace-nowrap" ("opam install " ^ package.name ^ "." ^ specific_version) %>
+  <%s! Copy_to_clipboard.small_code_snippet ~id:"install-package" ~class_:"whitespace-nowrap" ("opam install " ^ package.name) %>
```

Comment:
removing the version entirely only makes sense when we are on the latest version of the package, i.e. on any URL like this: `https://ocaml.org/p/PACKAGE_NAME/latest`.

Also, I think we don't need to remove the version from the `opam install` command because being precise here is actually helpful so people get an error message if they haven't updated their opam.

Reply:
It should now be omitted for only the dune dependency for latest version of the package. Am i correct?

Reply:
I don't know, there's no new commit on this branch yet.